packages/api/CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
 ## [UNRELEASED]
+## [1.3.0-alpha.1](https://github.com/cennznet/api.js/compare/release/1.2.1...prerelease/1.3.0-alpha.1) (23/10/2020)
+Polkadot's changelog at [2.3.1](https://github.com/polkadot-js/api/blob/master/CHANGELOG.md#231-oct-19-2020)
```

Comment:
Perhaps we need to mention the range of polkadot changelogs. Something like: [1.1.1-2.3.1]

Reply:
We plan to have [how to migrate to 1.3.x](url) page before the actual release which will talk more about things that have changed from 1.1.1 to 2.3.1